code/src/layouts/app-header/AppHeader.tsx

```diff
@@ -0,0 +1,13 @@
+import Header from "@/layouts/app-header/components/header/Header";
+import Categories from "@/layouts/app-header/components/categories/Categories";
+
+const AppHeader = () => {
+  return (
+    <>
+      <Header />

```

Comment:
Maybe it is not the best name for this component. I suggest renaming `AppHeader` to `Header` and `Header` to something like `HeaderToolbar`. Also we can rename Categories to `HeaderCategories` to make it clear that it belongs to `Header`

Reply:
> Maybe it is not the best name for this component. I suggest renaming `AppHeader` to `Header` and `Header` to something like `HeaderToolbar`. Also we can rename Categories to `HeaderCategories` to make it clear that it belongs to `Header`

It passed to `layouts` folder And naming for `layouts` start with `app` like `app-wrapper` etc. 
It just our code conventions

Reply:
No, we discussed that we should prefix with `App` only wrapper from mui components.